src_rebuild/GAME/C/SYSTEM.C

```diff
@@ -1,12 +1,14 @@
 #include "DRIVER2.H"
 
 #ifndef PSX
+#include <stdint.h>
```

Comment:
because else we'll get the following errors:
```
In file included from /usr/include/c++/10.2.0/cstdlib:75,
                 from /usr/include/c++/10.2.0/stdlib.h:36,
                 from /usr/include/SDL2/SDL_stdinc.h:40,
                 from /usr/include/SDL2/SDL_main.h:25,
                 from /usr/include/SDL2/SDL.h:32,
                 from GAME/C/SYSTEM.C:5:
/usr/include/stdlib.h:425:5: error: ‘int32_t’ does not name a type
  425 |     int32_t *fptr;  /* Front pointer.  */
      |     ^~~~~~~
/usr/include/stdlib.h:426:5: error: ‘int32_t’ does not name a type
  426 |     int32_t *rptr;  /* Rear pointer.  */
      |     ^~~~~~~
/usr/include/stdlib.h:427:5: error: ‘int32_t’ does not name a type
  427 |     int32_t *state;  /* Array of state values.  */
      |     ^~~~~~~
/usr/include/stdlib.h:431:5: error: ‘int32_t’ does not name a type
  431 |     int32_t *end_ptr;  /* Pointer behind state table.  */
      |     ^~~~~~~
/usr/include/stdlib.h:435:8: error: ‘int32_t’ has not been declared
  435 |        int32_t *__restrict __result) __THROW __nonnull ((1, 2));
      |        ^~~~~~~
```